docs/md-loader.js

```diff
@@ -1,23 +1,47 @@
+// @ts-check
 const GrayMatter = require('gray-matter');
+const Unified = require('unified');
+const RemarkParse = require('remark-parse');
+const Gfm = require('remark-gfm');
+const RemarkRehype = require('remark-rehype');
+const RehypeStringify = require('rehype-stringify');
+const RehypeRaw = require('rehype-raw');
```

Comment:
Really interesting to see PascalCasing in Javascript code, I guess it's just a habit derived from Hapi.js :wink:

Reply:
Yes, it is :D I also wrote a custom eslint rule for this: https://github.com/typeofweb/eslint-plugin/blob/b862c2cdac0549166ea685e36617ee6c634f0270/capitalize-modules.js